Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env groovy
+
+properties([
+    [$class: 'jenkins.model.BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '5']],
+    pipelineTriggers([cron('@hourly')]),
+])
+
+node('docker') {
+    timestamps {
+        docker.image('debian').inside('-u 0:0') {
+            stage('Prepare Container') {
+                sh 'apt-get update -qy && apt-get install -qy --force-yes curl'
```

Comment:
Better to start with a container that already has these tools installed, like `cloudbees/java-build-tools` off the top of my head.